client/src/store/reducers/ui/typography.js

```diff
@@ -2,33 +2,42 @@ import { handleActions } from 'redux-actions';
 
 const initialState = {
   font: 'serif',
-  size: 3,
-  sizeMax: 5,
-  sizeMin: 0
+  fontSize: {
+    current: 3,
+    max: 5,
+    min: 0
+  },
+  margins: {
+    current: 1,
+    max: 3,
+    min: 0
+  }
 };
 
 const selectFont = (state, action) => {
   return Object.assign({}, state, { font: action.payload });
 };
 
-const incrementFontSize = (state) => {
-  let value = state.size;
-  if (value < state.sizeMax) {
-    value = value + 1;
+const incrementAttribute = (state, attribute) => {
+  const parameter = state[attribute];
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).